flink-runtime/src/main/java/org/apache/flink/runtime/rpc/jobmaster/JobMaster.java

```diff
@@ -135,12 +256,12 @@ public void registerAtResourceManager(final String address) {
 	 *
 	 * @param jobMasterRegistration Job master registration info which is sent to the resource
 	 *                              manager
-	 * @param attemptNumber Registration attempt number
+	 * @param attemptNumber         Registration attempt number
 	 * @param resourceManagerFuture Future of the resource manager gateway
-	 * @param registrationRun UUID describing the current registration run
-	 * @param timeout Timeout of the last registration attempt
-	 * @param maxTimeout Maximum timeout between registration attempts
-	 * @param deadline Deadline for the registration
+	 * @param registrationRun       UUID describing the current registration run
+	 * @param timeout               Timeout of the last registration attempt
+	 * @param maxTimeout            Maximum timeout between registration attempts
+	 * @param deadline              Deadline for the registration
 	 */
 	void handleResourceManagerRegistration(
```

Comment:
The differences made here is due to the automatic code reformatting. I focused on the implementation of leader election in this PR. Hence, I did not delete your code about registration here. 

Since the code will not be used in the following PR, I will delete it :)